onpay.php

```diff
@@ -425,10 +470,37 @@ private function getPaymentWindow($order, $payment, $currency) {
         return $paymentWindow;
     }
 
+    /**
+     * Method used for setting a payment info parameter. The value is attempted set, if this fails we'll ignore the value and do nothing.
+     * $value can be a single value or an array of values passed on as arguments.
+     * Validation of value happens directly in the SDK.
+     *
+     * @param $paymentInfo
+     * @param $parameter
+     * @param $value
+     */
+    private function setPaymentInfoParameter(&$paymentInfo, $parameter, $value) {
+        if ($paymentInfo instanceof \OnPay\API\PaymentWindow\PaymentInfo) {
+            $method = 'set'.$parameter;
+            if (method_exists($paymentInfo, $method)) {
+                try {
+                    if (is_array($value)) {
+                        call_user_func_array([$paymentInfo, $method], $value);
+                    } else {
+                        call_user_func([$paymentInfo, $method], $value);
+                    }
+                } catch (\OnPay\API\Exception\InvalidFormatException $e) {
+                    // No need to do anything. If the value fails, we'll simply ignore the value.
+                }
+            }
+        }
+    }
+
     private function renderPaymentWindowForm(\OnPay\API\PaymentWindow $paymentWindow) {
         $this->smarty->assign(array(
             'form_action' => $paymentWindow->getActionUrl(),
             'form_fields' => $paymentWindow->getFormFields(),
+            'form_info_fields' => $paymentWindow->getInfo()->getFields(),
```

Comment:
This step should NOT be necessary, it already happens internally in the $paymentwindow->getFormFields() method